src/Session/profile.py

```diff
@@ -56,6 +61,14 @@ def listFoodStorages(self, foodStorage_type):
             return []
         return list(self._foodStorages[foodStorage_type].keys())
     
+    ''' Adds foods to the opened food storage that the user is in. 
+        Can add multiple food items. '''
```

Comment:
I think you should specify that `foods` in this case is an array, just for clarity. So something like `Adds array of foods to the opened food storage that the user is in. `